standard/ERCs/erc-6900.md

```diff
@@ -586,6 +587,19 @@ ERC-4337 compatible accounts must implement the `IAccount` interface, which cons
 
 This proposal includes several interfaces that build on ERC-4337. First, we standardize a set of modular functions that allow smart contract developers greater flexibility in bundling validation, execution, and hook logic. We also propose interfaces that provide methods for querying execution functions, validation functions, and hooks on a modular account. The rest of the interfaces describe a module's methods for exposing its modular functions and desired configuration, and the modular account's methods for installing and removing modules and allowing execution across modules and external addresses.
 
+### ERC 4337 Dependency
+
+ERC-6900's main objective is to create a secure and interoperable foundation through modular modules to increase the velocity and security of the Smart Account ecosystem and ultimately the wallet ecosystem.
+Currently, the ERC-6900 standard enforces the ERC-4337 for the validation and execution, however, this does not dictate that ERC-6900 will continue to be tied to the ERC-4337.
+It is likely that smart account builders will want to develop modular accounts that do not use ERC-4337 in the future, e.g., Native AA on rollups.
+Moreover, it is expected that the version of EntryPoint contract will continue to evolve until there is a protocol-level account abstraction.
```

Comment:
+1 on those points. 6900 accounts can be used with or without 4337 support. It should be able to support different flavors of AA.